### PR TITLE
Add vincbeck as committer

### PR DESCRIPTION
**Pull request by vincbeck**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by ephraimbuddy — APPROVED** [review]

File: docs/apache-airflow/project.rst [hunk 1]
```diff
@@ -87,6 +87,7 @@ Committers
 - Tomasz Urbaszek (@turbaszek)
 - Tzu-ping Chung (@uranusjr)
 - Vikram Koka (@vikramkoka)
+- Vincent Beck(@vincbeck)
```
Comment (hussein-awala) [inline docs/apache-airflow/project.rst]:
```suggestion
- Vincent Beck (@vincbeck)
```

**Review by hussein-awala — APPROVED** [review]

File: docs/apache-airflow/project.rst (hunk 1 above)
Comment (potiuk) [reply docs/apache-airflow/project.rst]: Oh god... We have ANOTHER committer list :)

Comment (potiuk) [reply docs/apache-airflow/project.rst]: @vincbeck - can you please add @mobuchowski and @phanikumv as well while correcting the space :). Pretty please :)

Comment (potiuk) [reply docs/apache-airflow/project.rst]: And I will correct instructions

Comment (potiuk) [reply docs/apache-airflow/project.rst]: Ah .. it was already there//

Comment (vincbeck) [reply docs/apache-airflow/project.rst]: > @vincbeck - can you please add @mobuchowski and @phanikumv as well while correcting the space :). Pretty please :)

Will do!

**Review by potiuk — APPROVED** [review]

File: docs/apache-airflow/project.rst (hunk 1 above)
Comment (potiuk) [reply docs/apache-airflow/project.rst]: :bow:

**Pull request merged**